aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -87,6 +87,13 @@ Tensor isnan(const Tensor& self) {
   return self != self;
 }
 
+Tensor isfinite(const Tensor& self) {
+  if (!self.is_floating_point()) {
+    return at::ones_like(self, at::kBool);
```

Comment:
Does it matter that we're not using `memory_format=torch.legacy_contiguous_format`?

Reply:
changed to `at::MemoryFormat::Preserve` after discussing with Vitaly